std_msgs/Empty empty
---
std_msgs/Float32MultiArray data